faker/providers/bank/fr_CH/__init__.py

```diff
@@ -0,0 +1 @@
+from ..de_CH import Provider  # noqa
```

Comment:
By not creating a subclass, the docstrings will be the same, and that will be reflected in the docs. If there is no difference, then it is preferable to be explicit about it in the docstrings too so that the docs will also include that information. Please create a subclass similar to what has been done [here](https://github.com/joke2k/faker/blob/9e360fa90eb6bfe3fde4c320ac32f033cc803e00/faker/providers/bank/fil_PH/__init__.py#L4).